src/context.test.js

```diff
@@ -0,0 +1,137 @@
+import { describe } from 'riteway';
+import dom from 'cheerio';
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+
+import { Provider, withFeatures, useFeatures } from './context';
+
+const render = ReactDOMServer.renderToStaticMarkup;
+
+describe('withFeatures()', async assert => {
```

Comment:
Let's move this set of tests to its own file to be consistent with the rest of the lib.